Functions.Templates/ProjectTemplate_v4.x/FSharp-Isolated/Company.FunctionApp.fsproj

```diff
@@ -1,17 +1,18 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>
```

Comment:
Shouldn't this also just be `TargetFrameworkValue`? I'm not sure why this was `net6.0` before...

Reply:
Yeah I am not sure. Tagging @liliankasem, @kshyju

Reply:
I haven't touched the proj version - @soninaren ?

Reply:
Traces back to https://github.com/Azure/azure-functions-templates/pull/1074.

The initial state was arguably wrong, and it was bumped once. I still assert that it should use the templating engine parameter for this.

Reply:
Got it. Updated to use the parameter.